code/game/objects/structures/chess.dm

```diff
@@ -0,0 +1,87 @@
+/* I don't know how to code.
+ * This is the only way I can contribute to the codebase with this shitty addition other than mapping.
+ * I don't know how to literally code chess into SS13, not even coding it so you can only do so and so moves, but I don't know how to even make a table to open a menu with little draggable pictures.
+ * Please forgive this shit code and the actual trash that this whole thing is, because adding something you can build in-game to have a little fun every 50-100 shifts or so is fun.
+ * If you are porting this, apologies, but you should check sheet_types.dm so these can be buildable at least.
+ */
+
+
+/obj/structure/chess
+	anchored = FALSE
+	density = FALSE
+	icon = 'icons/obj/chess.dmi'
+	icon_state = "singularity_s1"
+	name = "Singularity"
+	desc = "You've just been pranked by the Syndicate Chess Grandmaster! Report this to CentCom."
+	max_integrity = 100
+	resistance_flags = FLAMMABLE
+	var/buildstacktype = /obj/item/stack/sheet/metal
+	var/buildstackamount = 2
```

Comment:
Useless vars, these are not windows or chairs with subtypes made of different materials.